siddhi-extension-archetype/siddhi-map/src/main/resources/archetype-resources/component/src/main/java/__mapTypeInLowerCase__/sinkmapper/__classNameOfSinkMapper__.java

```diff
@@ -0,0 +1,155 @@
+package ${package}.${mapTypeInLowerCase}.sinkmapper;
+
+import org.wso2.siddhi.annotation.Example;
+import org.wso2.siddhi.annotation.Extension;
+import org.wso2.siddhi.core.config.SiddhiAppContext;
+import org.wso2.siddhi.core.event.Event;
+import org.wso2.siddhi.core.stream.output.sink.SinkListener;
+import org.wso2.siddhi.core.stream.output.sink.SinkMapper;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+import org.wso2.siddhi.core.util.transport.OptionHolder;
+import org.wso2.siddhi.core.util.transport.TemplateBuilder;
+import org.wso2.siddhi.query.api.definition.StreamDefinition;
+
+import java.util.Map;
+
+/**
+ * This is a sample class-level comment, explaining what the extension class does.
+ */
+
+/**
+ * Annotation of Siddhi Extension.
+ * <pre><code>
+ * eg:-
+ * {@literal @}Extension(
+ * name = "The name of the extension",
+ * namespace = "The namespace of the extension",
+ * description = "The description of the extension (optional).",
+ * //Sink configurations
```

Comment:
sink mapper configs. Check all other places

Reply:
Fixed in 624c01a